default.nix

```diff
@@ -1,15 +1,8 @@
 { pkgs ? import <nixpkgs>{}, versionArg ? "" } :
-let
-    inherit(pkgs)
-        buildGoModule
-        statik
-        git
```

Comment:
> doesn't L11 also need a change to ${pkgs.git}/bin/git rev-parse?

We don't need to in this case since git is specified as a build input and will be in the `PATH`

I'll update the script to use the one-liner